frontend/public/components/storage/create-pvc.tsx

```diff
@@ -266,16 +283,22 @@ export const CreatePVCForm: React.FC<CreatePVCFormProps> = (props) => {
         Volume Mode
       </label>
       <div className="form-group">
-        {volumeModeRadios.map((radio) => (
-          <RadioInput
-            {...radio}
-            key={radio.value}
-            onChange={handleVolumeMode}
-            inline
-            checked={radio.value === volumeMode}
-            name="volumeMode"
-          />
-        ))}
+        {storageProvisioner.includes(cephFSProvisionerSuffix) ? (
```

Comment:
you can use 1st index of this list.

https://github.com/openshift/console/blob/eec5efd40c79b95ebf3ba4315d5719b197840aa9/frontend/packages/console-shared/src/utils/storage-utils.ts#L6

```suggestion
        {storageProvisioner.includes(cephFSProvisionerSuffix) ? (
```

Reply:
done